### ISSUE TITLE
Help needed.

### ISSUE THREAD

**Issue by Zibri** [Zibri/cloudflare-cors-anywhere#18]
Hello! I need help with this utility.

It works, but if I request a page from another cloudlfare hosted service, I get the cloudflare captcha. It works in the cloudflare dash but it does not work outside of it.

Do you have any idea how to avoid it?

Example url: https://feedly.com/v3/streams/contents?streamId=feed/http://googlechromereleases.blogspot.com/feeds/posts/default

**Comment by Zibri** [comment]
@maurojs10 any idea?

**Comment by maurojs10** [comment]
@Zibri I have no idea. I'm sorry.

**Comment by tiagozip** [comment]
i don't think this is possible to solve since you can't just bypass cloudflare's captcha
